### PR TITLE
Adjust 'change' to 'fix' (#8664)

### PR DESCRIPTION
**Pull request by tomlankhorst**
[ci skip]

@Gymnasiast #8664

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -63,6 +62,7 @@
 - Fix: [#8585] Part of track missing on air powered vertical coaster.
 - Fix: [#8588] Guest list scrolling breaks above ~2000 guests.
 - Fix: [#8591] Game loop does not run at a consistent tick rate of 40 Hz.
+- Fix: [#8647] Increased 'guest generation probability' thresholds of marketing campaigns.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]:
```suggestion
- Fix: [#8647] Increased 'guest generation probability' thresholds of marketing campaigns (original bug). 
```

**Comment by Gymnasiast** [comment]
To avoid AppVeyor firing, the `[ci skip`] needs to be in the commit _title_, not the message. This is incorrect behaviour of AppVeyor, but we have to deal with it unfortunately.

**Comment by tomlankhorst** [comment]
> To avoid AppVeyor firing, the `[ci skip`] needs to be in the commit _title_, not the message. This is incorrect behaviour of AppVeyor, but we have to deal with it unfortunately.

Lovely

File: distribution/changelog.txt [hunk 2]
```diff
@@ -63,6 +62,7 @@
 - Fix: [#8585] Part of track missing on air powered vertical coaster.
 - Fix: [#8588] Guest list scrolling breaks above ~2000 guests.
 - Fix: [#8591] Game loop does not run at a consistent tick rate of 40 Hz.
+- Fix: [#8647] Increased 'guest generation probability' thresholds of marketing campaigns (original bug).
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: Oh ****, forgot to look at the message proper. This is the last time, I promise! (Reason for change: we always tell which bug is fixed, not _how_ it was fixed.)
```suggestion
- Fix: [#8647] Marketing campaigns check for entry fees below £1 (original bug). 
```

**Comment by tomlankhorst** [comment]
Thanks for your time

**Comment by Gymnasiast** [comment]
Thanks for _your_ time too.

**Pull request merged**